src/include/OpenImageIO/unordered_map_concurrent.h

```diff
@@ -74,13 +74,16 @@ OIIO_NAMESPACE_BEGIN
 ///
 
 template<class KEY, class VALUE, class HASH=std::hash<KEY>,
-         class PRED=std::equal_to<KEY>, size_t BINS=16,
+         class PRED=std::equal_to<KEY>, int LOG2_BINS=4,
```

Comment:
I think this is a confusing change. Can we keep it BINS, use static_assert to ensure it's a power of 2 (if we must), and figure out the bin shift via constexpr and bit twiddling tricks?

Reply:
I can give it a try ...